dev-libs/hyperscan/hyperscan-5.2.1.ebuild

```diff
@@ -3,9 +3,9 @@
 
 EAPI=7
 
-PYTHON_COMPAT=( python2_7 )
+PYTHON_COMPAT=( python3_{6..8} )
 
-inherit cmake python-r1 flag-o-matic
+inherit cmake python-any-r1 flag-o-matic
```

Comment:
Please sort alphabetically where possible, these shouldn't cause conflicts.

Reply:
Python is not required here either.

Reply:
Python is required as a build time dependency, see this https://github.com/intel/hyperscan/blob/d79973efb1fcf5ed338122882c1f896829767fb6/CMakeLists.txt#L75-L82